12
OK, at least one conversion triggered the 'TruncatedInteger' exception :)
That's all, folks!
